tests/integration/public/auth/test_auth_routes.py

```
@@ -43,6 +46,42 @@ def auth_app(monkeypatch: pytest.MonkeyPatch) -> Flask:
+def auth_db_client(monkeypatch: pytest.MonkeyPatch):
+    """Flask test client with real DB for testing auth callbacks."""
+    monkeypatch.setenv("FLASK_SECRET_KEY", "testing-secret")
+
+    cookie = types.SimpleNamespace(
+        name="uid_enc_copy",
+        httponly=True,
+        secure=False,
+        samesite="Lax",
+        max_age=3600,
+    )
+    oauth_cfg = types.SimpleNamespace(consumer_key="key", consumer_secret="secret")
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "consumer_secret"

(S106)